lib/src/obsoleted/rules/no_magic_number_rule.dart

```diff
@@ -8,6 +8,8 @@ import '../../utils/node_utils.dart';
 import '../../utils/rule_utils.dart';
 import 'obsolete_rule.dart';
 
+const _defaultMagicNumbers = [-1, 0, 1];
```

Comment:
Let's keep the variable as a static variable of the rule class.

Reply:
fixed